libdevcore/JsonUtils.h

```diff
@@ -24,4 +24,19 @@ namespace dev
 {
 // Throws UnknownField() if _obj contains field names not listed in _allowedFields.
 void validateFieldNames(json_spirit::mObject const& _obj, std::set<std::string> const& _allowedFields);
+
+// Converts json value type to string
+std::string jsonTypeAsString(json_spirit::Value_type _type);
+
+// Check json _o with validation map that reuires certain field of certain type to be present in
+// json
+enum jsonField
+{
+    Required,
+    Optional
+};
+using jsonTypeSet = std::set<json_spirit::Value_type>;
+using jsonType = std::pair<jsonTypeSet, jsonField>;
+void requireJsonFields(json_spirit::mObject const& _o, std::string const& _config,
```

Comment:
The function does a little bit different job from what "require Json Fields" suggests.

Please document the function.

Reply:
could you clarify what "document the function" should look like ?